src/main/java/bio/terra/cli/context/utils/Printer.java

```diff
@@ -14,23 +14,23 @@
 public class Printer {
   private static final org.slf4j.Logger logger = LoggerFactory.getLogger(Printer.class);
 
-  public static final PrintWriter DEFAULT_OUT_WRITER = getPrintWriter(System.out);
-  public static final PrintWriter DEFAULT_ERR_WRITER = getPrintWriter(System.err);
+  public static final PrintStream DEFAULT_OUT_STREAM = System.out;
+  public static final PrintStream DEFAULT_ERR_STREAM = System.err;
 
-  private final PrintWriter outWriter;
-  private final PrintWriter errWriter;
+  private final PrintStream out;
+  private final PrintStream err;
```

Comment:
What's up with having `public DEFAULT_OUT_STREAM` and `getOut` from the singleton? Are these always returning the same thing? I feel like I'm missing something.

Reply:
Ah oversight on my part. Changed to public. `getOut` will always return the default value at the moment, but we may change this in the future (e.g. with a `--output-to-file` option on all commands). This class was created centralize the hard-coding of `System.out`, which was done in several places previously.